### PR TITLE
Add support for websocket endpoint with graphql-ws

### PR DESCRIPTION
**Pull request by lethot**
Changes proposed in this pull request:

- add graphql-ws to support subscription and full graphql over websockets

### CODE REVIEW

**Comment by ntziolis** [comment]
When can we expect this to be merged in?

**Comment by ardatan** [comment]
@lethot We can have more features by adding GraphQL Tools Url Loader as in this PR. What do you think @acao ?
https://github.com/contrawork/graphql-helix/pull/33/files#diff-d04dada0c5eb3a6ab96feb1d2ce9abc653a7b5d859693aea37621e43a6b4b9feR3

**Comment by maxpain** [comment]
Any updates?

File: packages/graphql-playground-react/src/components/EndpointPopup.tsx [hunk 1]
```diff
@@ -20,7 +22,37 @@ export interface State {
...
+              this.setState({ valid: true })
+              unsubscribe()
+            },
+            error: () => {
+              this.setState({ valid: false });
+              unsubscribe()
+            },
+            complete: () =>  {},
+          },
+        );
```
Comment (enisdenjo) [inline packages/graphql-playground-react/src/components/EndpointPopup.tsx:52]: A simpler way to achieve the same check would be by disabling `lazy` mode (connect on instantiation) and check if the client `connected`.

```suggestion
        const client = createClient({
          url: this.state.endpoint,
          retryAttempts: 0,
          lazy: false,
          on: {
            closed: () => this.setState({ valid: false }),
            connected: (socket) => {
              this.setState({ valid: true });
              socket.close(1000); // close after validation
            },
          },
        });
```

**Comment by maxpain** [comment]
Any news?

**Review by AlexKutschera — APPROVED** [review]

**Comment by elderapo** [comment]
What's blocking from getting this merged? Any way to help?

**Review by acao — APPROVED** [review]
Looks good to me!

File: packages/graphql-playground-react/src/components/EndpointPopup.tsx (hunk 1 above)
Comment (acao) [reply packages/graphql-playground-react/src/components/EndpointPopup.tsx:52]: oof, I forgot to commit this suggestion. @lethot or @enisdenjo can you follow on a PR with this correction?

**Comment by intellix** [comment]
This says that it was merged on 25th April but I don't see any newly published graphql-playground-html packages since 1yr ago: https://www.npmjs.com/package/graphql-playground-html (1.6.30 @ 11/4/2021 6pm).

Is this merged item available anywhere?

**Pull request merged**